src/XMakeTasks/Microsoft.CSharp.targets

```diff
@@ -17,6 +17,15 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <!-- 
+        We are doing a cross-targeting traversal build if there is no list of target frameworks specified
```

Comment:
If there's a design doc on some github SDK page about what a traversal build is, this comment would be a good spot to reference it in case users want to understand the bigger picture. :)